agency/README.md

```diff
@@ -241,24 +242,24 @@ Query Params:
+| `service_start`      |                                                         | Vehicle introduced into service at the beginning of the day (if program does not operate 24/7) | `unavailable`                                      | `available`         | Vehicle is on the street and available for customer use.                |
+| `service_end`        | `low_battery`, `maintenance`, `compliance`, `off_hours` | A vehicle is no longer available due to `event_reason`                                         | `available`                                        | `unavailable`       |                                                                         |
+| `provider_drop_off`  |                                                         | Vehicle moved for rebalancing                                                                  | `removed`, `elsewhere`                             | `available`         |                                                                         |
+| `provider_pick_up`   | `rebalance`, `maintenance`, `charge`, `compliance`      | Vehicle removed from street and will be placed at another location to rebalance service        | `available`, `unavailable`, `elsewhere`            | `removed`           |                                                                         |
+| `city_pick_up`       |                                                         | Vehicle removed by city                                                                        | `available`, `unavailable`                         | `removed`           |                                                                         |
+| `reserve`            |                                                         | Customer reserves vehicle                                                                      | `available`                                        | `reserved`          | Vehicle is reserved or in use.                                          |
+| `cancel_reservation` |                                                         | Customer cancels reservation                                                                   | `reserved`                                         | `available`         |                                                                         |
+| `trip_start`         |                                                         | Customer starts a trip                                                                         | `available`, `reserved`                            | `trip`              |                                                                         |
+| `trip_enter`         |                                                         | Customer enters a service area managed by agency during an active trip.                        | `removed`, `elsewhere`                             | `trip`              |                                                                         |
```

Comment:
This should be clarified from `service_area` to `geographic boundary` or similar, ie, we don't want events when you leave all the specific `service_areas`, but rather, just when you leave / end municipal boundary.

Reply:
I believe we have a solid boundary definition in provider now, take a look for guidance.

Reply:
@hunterowens agree the intent is not to receive events on entry and lead from each service area, but actually the boundary that the agency is responsible for managing.  I do think that `service_areas` are meant to define that boundary and that should be the single source of truth.  I will modify to clarify that crossing into or out of the region that is the union of all `service_areas` is the desired trigger.

Reply:
In the case where an agency doesn't want to allow providers to operate in all areas of their jurisdiction, I wouldn't necessarily expect `service_areas` to cover the whole jurisdiction but I'd definitely expect providers to include events that happen in those prohibited areas. It's probably worth either distinguishing service areas from jurisdictional boundaries or clarifying that agencies are expected to include either `unrestricted` or `restricted` areas covering their full jurisdiction in the `service_areas` endpoint.

Reply:
The former makes more sense to me—it seems unnecessarily onerous to make agencies carve up their whole city boundary into restricted and unrestricted zones to serve in the API, and also I would imagine providers would rather have a stable concept of a given city's boundaries instead of having to trust that the agency will always give them a set of service areas that add up to a consistent boundary.

Reply:
@hunterowens I resolved the wording to be consistent with Provider. @rf you are bringing up some good points and needed clarifications as to what constitutes `service_areas`.  I would like to continue this discussion in a separate issue that I will open, since the purpose of the PR is to clarify and make some fixes to the event model.

Reply:
Sounds good, thanks! If it wasn't clear, the reason I mentioned those issues here was just to express support for using language that is about the municipal area and not `service_areas`; the update you just made looks good to me. Happy to hash out remaining issues with `service_areas` elsewhere.